### DISCUSSION TITLE
Getting started?

### DISCUSSION THREAD

**Discussion by NKT00** [nicolai-wachenschwan/arc-overhang-prusaslicer-integration#51] [General]
How do I get this to work? Python is installed, full path is set into the post-process box, along with the path to the script.
After a bit of messing, I finally got to:

Post-processing script C:\Users\Nigel\AppData\Local\Programs\Python\Python311\python.exe C:\Users\Nigel\Downloads\prusa_slicer_post_processing_script.py on file C:\Users\Nigel\AppData\Local\Temp\.21808.gcode.pp failed.
Error code: 1

Anyone know what this means?

Edit: digging into temp, the gcode from Prusaslicer gets copied there, but it doesn't get modified. Possibly some write permission thing?

**Comment by mam326** [comment]
I have the exact same error whilst exporting the g-code.

![error-code-1 - Copy](https://github.com/nicolai-wachenschwan/arc-overhang-prusaslicer-integration/assets/65680273/8afbe6dc-40e5-4a1c-a87f-a619f34c2480)

**Reply by bettsg** [reply]
Old question, but figured I'd answer since I was supremely frustrated for the last while with that same error until I found the answer a couple posts down. Don't use python.exe. You should have a C:\Windows\py.exe
